### DISCUSSION TITLE
Feature request: vim-tmux-navigator replacement

### DISCUSSION THREAD

**Discussion by mleveck** [ghostty-org/ghostty#3607] [Feature Requests, Ideas] [answered]
I think a lot of what I and others use tmux for can be replaced by simply using Ghostty's excellent key bind configurations. Doing so gives us more seamless access to Ghostty's features, without fighting through tmux.

But one feature I don't think we can get at the moment is seamless navigation between vim and terminal splits/panes, that we can get with [vim-tmux-navigator](https://github.com/christoomey/vim-tmux-navigator).   I don't think it's really possible to implement this without code level changes to Ghostty.  So I'm adding it here as a feature request.

**Comment by mitchellh** [comment, marked as answer]
Requires: #601 

Closing until then. You can add your request in there.

**Reply by mitchellh** [reply]
Sorry that turned into #2353
